andela/thorin-rc
# #157913992 Setup TravisCI

#### What does this PR do?
Setup TravisCI for continuous integration.
#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Create a branch and push commit to repo, then view build log on [TravisCI](https://travis-ci.org/andela/thorin-rc).
Make a pull request then view TravisCI check.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#157913992
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
